pkg/cassandra/config/config.go

```diff
@@ -143,8 +143,9 @@ func (c *Configuration) NewCluster(logger *zap.Logger) (*gocql.ClusterConfig, er
 
 	if c.Authenticator.Basic.Username != "" && c.Authenticator.Basic.Password != "" {
 		cluster.Authenticator = gocql.PasswordAuthenticator{
-			Username: c.Authenticator.Basic.Username,
-			Password: c.Authenticator.Basic.Password,
+			Username:              c.Authenticator.Basic.Username,
+			Password:              c.Authenticator.Basic.Password,
+			AllowedAuthenticators: c.Authenticator.AllowedAuthenticators,
 		}
```

Comment:
The reason this is getting flagged as not covered is apparently because we do not use auth in the integration tests, so this path is never exercised. It would be a good idea to change the integration tests to actually use a password, if you want to book a ticket for that.

<img width="1120" alt="image" src="https://github.com/jaegertracing/jaeger/assets/3523016/f5d14b1a-b7fe-4520-91bc-3cae7276558d">